pkg/shortnames/shortnames.go

```diff
@@ -386,6 +380,39 @@ func Resolve(ctx *types.SystemContext, name string) (*Resolved, error) {
 	return resolved, nil
 }
 
+// Prompt opens a prompt in the terminal to let the user choose *one* of the
+// specified items.
```

Comment:
“… of the specified fully-qualified image references”, or something like that? It’s not a general-purpose prompting utility.